### ISSUE TITLE
 Translation problem in PropertyGroups with "group.display_name"

### ISSUE THREAD

**Issue by hauzmann** [OFFLINE-GmbH/oc-mall-plugin#547]
It appears that {{ group.display_name }} translation in ProductsFilter component somehow doesn't work.

I filled all necessary data in "PropertyGroups > Edit Group" backend tab, both for `display_name` and `name` fields in Russian and in English. But on the frontend when I switch language translation doesn't happen just for this field.

When I change code in `plugins/offline/mall/components/productsfilter/propertygroups.htm` from
`<div class="mall-property-group__name">{{ group.display_name }}</div>`
to
`<div class="mall-property-group__name">{{ group.name }}</div>`
translation works just fine.

**Comment by tobias-kuendig** [comment]
Could you try this change around here and see if this fixes things:

https://github.com/OFFLINE-GmbH/oc-mall-plugin/blob/3ea133894b99eeb2502be109e07f7c0c48f2b6d0/models/PropertyGroup.php#L100

```php
    public function getDisplayNameAttribute()
    {
        if ($this->getOriginal('display_name')) {
            // This changed.
            return $this->display_name;
        }

        return $this->name;
    }
```

**Comment by hauzmann** [comment]
Unfortunately, it does't help. Breaks down with en error.
[ERROR.pdf](https://github.com/OFFLINE-GmbH/oc-mall-plugin/files/5007806/ERROR.pdf)

**Comment by hauzmann** [comment]
If I entirely remove
  ```
  public function getDisplayNameAttribute()
    {
        if ($this->getOriginal('display_name')) {
            return $this->getOriginal('display_name'); 
        }

        return $this->name;
    }
```
from `oc-mall-plugin/models/PropertyGroup.php`,
`display_name` translation seems to work fine.

**Comment by tobias-kuendig** [comment]
This is fixed in 4dee13f, thank you @hauzmann!
